activities/Clock.activity/icons/before.svg

```diff
@@ -0,0 +1,7 @@
+<svg id="SvgjsSvg1001" width="288" height="288" xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:svgjs="http://svgjs.com/svgjs"><defs id="SvgjsDefs1002"></defs><g id="SvgjsG1008" transform="matrix(0.9167,0,0,0.9167,11.995172036743156,11.995200000000011)"><svg xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 445.462 445.462" viewBox="0 0 445.462 445.462" width="288" height="288"><path d="M380.225,65.236C338.157,23.168,282.225,0,222.731,0S107.305,23.168,65.237,65.236S0,163.237,0,222.731
```

Comment:
Icons must not be colored. You should use a more explicit name for the file.
Both icons should be identical except for the seconds hand.